src/dss/logs.py

```diff
@@ -0,0 +1,76 @@
+from lightkube import ApiError, Client
+from lightkube.resources.apps_v1 import Deployment
+from lightkube.resources.core_v1 import Pod
+
+from dss.config import DSS_NAMESPACE, MLFLOW_DEPLOYMENT_NAME
+from dss.logger import setup_logger
+
+# Set up logger
+logger = setup_logger("logs/dss.log")
+
+
+def get_logs(parts: str, name: str, lightkube_client: Client) -> None:
+    """
+    Retrieve logs from specified parts of the DSS application.
+
+    Args:
+        parts (str): Specify which part's logs to retrieve: 'notebooks' or 'mlflow'.
+        name (str): Name of the notebook or deployment to retrieve logs from.
+        lightkube_client (Client): The Kubernetes client.
+
+    Returns:
+        None
+    """
+    try:
+        if parts == "notebooks":
+            deployment = lightkube_client.get(Deployment, name=name, namespace=DSS_NAMESPACE)
+        elif parts == "mlflow":
+            deployment = lightkube_client.get(
+                Deployment, name=MLFLOW_DEPLOYMENT_NAME, namespace=DSS_NAMESPACE
+            )
+        else:
+            logger.error("Invalid value for 'parts'. Use 'notebooks' or 'mlflow'.")
+            return
+    except ApiError:
+        deployment = None
+
+    # The get command can return empty DeploymentList object with empty spec field
+    if deployment is None or deployment.spec is None:
+        if parts == "notebooks":
+            logger.error(f"No notebook found with the name '{name}'.")
+            logger.info(
+                "Please wait a moment or recreate the notebook with 'dss create-notebook'."
```

Comment:
Ideally we'd have a command that would list the notebooks and show their status(?), that way in this error msg we can have something like "Verify the notebook is running with <dss command> or recreate the notebook...". It doesn't have to be part of this PR, but something that can provide a better UX. What do you think?